### PR TITLE
Update website address and hex config

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ title: Introduction
 order: 0
 ---
 
-This is the guide to using the [Apollo](http://apollostack.com) JavaScript GraphQL client with the [Angular](https://angular.io) rendering library.
+This is the guide to using the [Apollo](http://apollodata.com) JavaScript GraphQL client with the [Angular](https://angular.io) rendering library.
```
Comment (stubailo) [inline source/index.md]: these should all be dev.apollodata.com, which is the homepage for Apollo Client.

File: source/index.md [hunk 2]
```diff
@@ -34,12 +34,12 @@ Angular Apollo provides a idiomatic Angular API, and is known to work in a strai
...
-If you want to use Apollo with a different front-end environment (for instance a different JavaScript framework, or a native mobile app), it is possible too! Please check out the links at the top of this document, or head to our [developer site](http://apollostack.com) to get started.
+If you want to use Apollo with a different front-end environment (for instance a different JavaScript framework, or a native mobile app), it is possible too! Please check out the links at the top of this document, or head to our [developer site](http://apollodata.com) to get started.
 
 <h2 id="learn-more">Learn More</h2>
 
 To learn more about Apollo, and how to use it in Angular, visit:
 
 - [GraphQL.org](http://graphql.org) for an introduction to GraphQL,
-- [Our website](http://www.apollostack.com/) to learn about the Apollo stack,
+- [Our website](http://www.apollodata.com/) to learn about the Apollo stack,
```
Comment (stubailo) [inline source/index.md]: This shouldn't say "Apollo Stack" anymore.

File: source/index.md (hunk 1 above)
Comment (kamilkisiela) [inline source/index.md]: @stubailo done

File: source/index.md (hunk 2 above)
Comment (kamilkisiela) [inline source/index.md]: @stubailo done

**Comment by stubailo** [comment]
I think all URLs should be `dev.apollodata.com` since `apollodata.com` is not a useful landing page for Apollo developers.

**Comment by kamilkisiela** [comment]
Done

**Comment by stubailo** [comment]
Thanks!

**Pull request merged**